pallets/estate/src/lib.rs

```diff
@@ -668,6 +674,102 @@ pub mod pallet {
 				},
 			)
 		}
+
+		#[pallet::weight(T::WeightInfo::destroy_estate())]
+		pub fn destroy_estate(
```

Comment:
Shall we call it dissolve_estate?

Reply:
Dissolve estate will remove the formation of estate back to number of land units.